support-frontend/assets/pages/digital-subscription-checkout/components/digitalCheckoutFormGift.tsx

```diff
@@ -277,7 +277,7 @@ function DigitalCheckoutFormGift(props: PropTypes): JSX.Element {
 						errorReason={props.submissionError}
 						errorHeading={submissionErrorHeading}
 					/>
-					<EndSummaryMobile orderIsAGift />
+					<EndSummaryMobile />
```

Comment:
`orderIsAGift` no longer needs to be passed as a prop to `EndSummaryMobile`, this information is retrieved from Redux when we `mapStateToProps` for the `EndSummary`.